Birthday-2024-Project/Resources/CampaignLevels/CampaignLevel1.tres

```diff
@@ -1,6 +1,8 @@
-[gd_resource type="Resource" script_class="LevelSetup" load_steps=3 format=3 uid="uid://b2tco3xr6sxna"]
+[gd_resource type="Resource" script_class="LevelSetup" load_steps=5 format=3 uid="uid://b2tco3xr6sxna"]
 
 [ext_resource type="Script" path="res://Scripts/LevelSetup.gd" id="1_14d3v"]
+[ext_resource type="Texture2D" uid="uid://cx4qtp5plvyyy" path="res://Art/Emotes/comfy.png" id="1_kn4ei"]
```

Comment:
emotes as previews look very crusty, I'm for making PNG previews for campaign just like for Sapling Levels

Reply:
These are just placeholder until the proper level previews are added